src/iothread.c

```diff
@@ -624,6 +660,22 @@ void IOThreadAfterSleep(struct aeEventLoop *el) {
     atomicSetWithSync(t->running, 1);
 }
 
+/* This is our io thread timer interrupt, called (1000/IO_THREAD_CRON_TIME) times per second.
+ * The current responsibility is to detect clients that have been stuck in the
+ * iothread for too long and hand them over to the main thread for handling. */
+
+ int ioThreadCron(struct aeEventLoop *eventLoop, long long id, void *clientData) {
+    UNUSED(eventLoop);
+    UNUSED(id);
+    IOThread *t = clientData;
+
+    if (listLength(t->clients) != 0) {
+        IOThreadClientsCron(t);
+    }
```

Comment:
maybe just IOThreadClientsCron(t) like main thread
```suggestion
    IOThreadClientsCron(t);
```